index.js

```diff
@@ -54,23 +61,22 @@ const resolver = Consumer.create({
 
     try {
       const getCaseResponse = await getCase.fetch();
-      const icwID = getCaseResponse.caseId;
+      let caseID = getCaseResponse.caseId;
 
       if (!getCaseResponse.exists) {
         const data = await submitCase.save();
-        const caseID = data.createcaseresponse.caseid;
+        caseID = data.createcaseresponse.caseid;
 
         logger.info({ caseID, message: 'Casework submission successful' });
-
-        return submitAudit({ success: true, caseID });
+        return submitAudit('resolver', { success: true, case_ID: caseID, externalID: externalId});
```

Comment:
You could just rename `externalId` to `externalID` and then this could just be 
```
return await submitAudit('resolver', { success: true, caseID, externalID });
```